CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.11.0
```

Comment:
`0.10.1` or  `0.10.0+1` if you want to make a patch version as per:
> Although semantic versioning doesn’t promise any compatibility between versions prior to 1.0.0, the Dart community convention is to treat those versions semantically as well. The interpretation of each number is just shifted down one slot: going from 0.1.2 to 0.2.0 indicates a breaking change, going to 0.1.3 indicates a new feature, and going to 0.1.2+1 indicates a change that doesn’t affect the public API.

From: https://dart.dev/tools/pub/versioning#semantic-versions

-------
IMO, that's nit picking, `0.10.1` or `0.10.0+1` makes no difference to me. It depends on whether you think it a minor or patch version change. This could be either :)

Reply:
I went with `0.10.1` - thx